TLM/TLM/State/Options.cs

```diff
@@ -159,13 +159,13 @@ public enum PersistTo {
 
         internal static int getRecklessDriverModulo() {
             switch (recklessDrivers) {
-                case 0:
+                case RecklessDrivers.PathOfEvil:
                     return 10;
-                case 1:
+                case RecklessDrivers.RushHour:
                     return 20;
-                case 2:
+                case RecklessDrivers.MinorComplaints:
                     return 50;
-                case 3:
+                case RecklessDrivers.HolyCity:
                     return 10000;
             }
             return 10000;
```

Comment:
Off-topic, but.... convert to switch expression?

```csharp
internal static int RecklessDriverModulo(RecklessDrivers level) => level switch
{
    RecklessDrivers.PathOfEvil => 10,
    RecklessDrivers.RushHour => 20,
    RecklessDrivers.MinorComplaints => 50,
    RecklessDrivers.HolyCity => 10000,
    _ => 10000,
};
```

Have an `int` field to cache the value (defaulting to holy city modulo) and update it whenever the option changes (via the option change handler)? Could probably also cache result of `IsDynamicLaneSelectionActive()` method in similar way? Would remove couple of function calls from what I assume are hotpath call sites...?

Reply:
interesting idea. but I worry my PR is getting too big.

Reply:
It's low priority, I'll create separate issue to track.

Reply:
I think performance is important. so many if conditions could be very punishing!